build/style_test.go

```diff
@@ -706,7 +687,10 @@ func TestStyle(t *testing.T) {
 			t.Fatal(err)
 		}
 
-		if err := stream.ForEach(filter, func(s string) {
+		if err := stream.ForEach(stream.Sequence(
+			filter,
+			stream.GrepNot(`: (field no|type No)Copy is unused \(U1000\)$`),
```

Comment:
@dominikh this might be a reasonable feature request for unused or lintutil - this type is never used, but this is by design; see the implementation and comments on it: https://github.com/cockroachdb/cockroach/blob/master/pkg/util/nocopy.go.